content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -192,6 +192,53 @@ The original destination type of the traffic is used to determine if a service o
 For instance, traffic which is addressed to a service, even though ultimately resolved to a pod IP, is always treated by the ambient mesh as to-service and would use a service-attached waypoint.
 {{< /tip >}}
 
+## Cross-namespace waypoint use {#usewaypointnamespace}
+
+Straight out of the box a waypoint proxy is usable by resources within the same namespace. Beginning with Istio 1.23 it is possible to use waypoints in different namespaces. In this section we will examine the gateway configuration required to enable cross-namespace use as well as how to configure your resources to use a waypoint from a different namespace.
+
+### Configure a waypoint for cross-namespace use
+
+In order to enable cross-namespace use of a waypoint, the `Gateway` should be configured to [allow routes](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.AllowedRoutes) from other namespaces.
+
+{{< tip >}}
+The keyword, `All`, may be specified as the value for `allowedRoutes.namespaces.from` in order to allow routes from any namespace.
+{{< /tip >}}
+
+The following Gateway would allow resources in a namespace called "cross-namespace-waypoint-consumer" to use this egress-gateway:
```

Comment:
```suggestion
The following `Gateway` would allow resources in a namespace called "cross-namespace-waypoint-consumer" to use this `egress-gateway`:
```

Reply:
`Gateway` makes sense to me but `egress-gateway` is not a resource type or primitive it's just a name. Would this be double-quoted instead of getting back-ticks?

Reply:
nm, we basically never make a distinction like that elsewhere